src/main/java/com/compass/vinyl/player/RecordPlayer.java

```diff
@@ -58,4 +60,8 @@
      */
     void deleteByTags(List<String> tags, RecordingConfig config);
 
+    default String getUniqueId(Scenario scenario, Serializer serializer) {
```

Comment:
getUniqueId doesn't seem to belong to the recordPlayer entity. It is more of a way to identify the scenario uniquely

Reply:
@vivek-commits then we should make `getUniqueId` instance method of `Scenario` class.

Reply:
@vivek-commits made the change